docs/deployment_and_config.md

```diff
@@ -99,8 +99,9 @@ all:
     redis_password: <redis_password>
 
     # OpenSearch Settings
-    opensearch_cluster_name: bap-opensearch
-    heap_size: 1g
+    opensearch_cluster_prefix: <opensearch_cluster_prefix>
+    opensearch_cluster_name: <opensearch_cluster_name>
+    heap_size: <heap_size>
```

Comment:
I wonder if this should be renamed to <opensearch_cluster_heap_size>

Reply:
Yes, we should rename it